frontend/package.json

```diff
@@ -3,15 +3,16 @@
     "version": "0.1.0",
     "private": true,
     "dependencies": {
-        "font-awesome": "4.7.0",
-        "@cthit/react-digit-components": "^1.0.0-beta.42",
         "@babel/runtime": "7.5.4",
+        "@cthit/react-digit-components": "^1.0.3",
```

Comment:
You need to update the dependencies that RDC v1.0.3 needs.

For example, redux is needed at all.